### PR TITLE
LUN-82: show correct if give up

### PR DESCRIPTION
**Pull request by SteffiStuffel**
This pull request belongs to an issue on our [bugtracker](https://issues.tuerantuer.org/secure/RapidBoard.jspa?rapidView=27&view=detail).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword _LUN_.

### CODE REVIEW

File: src/components/FeedbackSection.tsx [hunk 1]
```diff
@@ -60,15 +60,15 @@ const Feedback = ({ result, document, input, secondAttempt }: FeedbackPropsType)
...
 
   const message =
     result === 'correct'
       ? 'Toll, weiter so! \nDeine Eingabe ist richtig.'
       : result === 'incorrect' || !secondAttempt
-      ? `Schade, deine Eingabe ist falsch. Die richtige Antwort ist: ${
+      ? `${result === 'incorrect' ? `Schade, deine Eingabe ist falsch.` : ``} Die richtige Antwort ist: ${
           document?.article?.toLowerCase() === ARTICLES.diePlural ? 'die' : document?.article
         } ${document?.word}`
       : `Deine Eingabe ${input} ist fast richtig. Überprüfe Groß- und Kleinschreibung.`
```
Comment (klinzo) [inline src/components/FeedbackSection.tsx:74]: Maybe this should be split in serveral vars/conditions. Maybe same for above

**Review by klinzo — APPROVED** [review]

File: src/components/FeedbackSection.tsx (hunk 1 above)
Comment (SteffiStuffel) [reply src/components/FeedbackSection.tsx:74]: You mean the complete ? : - nesting?

Comment (klinzo) [reply src/components/FeedbackSection.tsx:74]: Yes, it's kind of confusing imo. But I don't really mind, so you can keep this as well.

Comment (SteffiStuffel) [reply src/components/FeedbackSection.tsx:74]: No, i agree, it is confusing.

File: src/components/FeedbackSection.tsx [hunk 2]
```diff
@@ -50,40 +36,30 @@ export interface FeedbackPropsType {
...
+  let Icon, background, message
+  if (result === 'correct') {
+    Icon = CorrectFeedbackIcon
+    background = correct_background
+    message = 'Toll, weiter so! \nDeine Eingabe ist richtig.'
+  } else if (result === 'incorrect' || result === 'giveUp' || !secondAttempt) {
+    Icon = IncorrectFeedbackIcon
+    background = incorrect_background
+    message = `Die richtige Antwort ist: ${
+      document?.article?.toLowerCase() === ARTICLES.diePlural ? 'die' : document?.article
```
Comment (klinzo) [inline src/components/FeedbackSection.tsx:48]: I think you don't need those non null checks:
```suggestion
      document.article.toLowerCase() === ARTICLES.diePlural ? 'die' : document.article
```

File: src/components/FeedbackSection.tsx (hunk 1 above)
Comment (SteffiStuffel) [reply src/components/FeedbackSection.tsx]: refactored it.

File: src/components/FeedbackSection.tsx (hunk 2 above)
Comment (SteffiStuffel) [reply src/components/FeedbackSection.tsx:48]: Ja document war in den Props optional, aber ich denke das ist nicht nötig / sinnvoll. Hab es mal unoptional gemacht.

**Pull request merged**